plugins/formal-verification/src/org/jetbrains/kotlin/formver/conversion/ProgramConversionContext.kt

```diff
@@ -134,14 +151,16 @@ class StmtConversionContext(val methodCtx: MethodConversionContext) {
     }
 }
 
+// Convert a statement, emitting the resulting Viper statements and
+// declarations into the context, returning a reference to the
+// expression containing the result.  Note that in the FIR, expressions
+// are a subtype of statements.
```

Comment:
Same as the comment [here](https://github.com/jesyspa/kotlin/pull/7/files#r1281617170).